packages/react/src/auto/shadcn/inputs/ShadcnAutoInput.tsx

```diff
@@ -15,39 +16,40 @@ export const makeShadcnAutoInput = ({
   Checkbox,
   Button,
 }: Pick<ShadcnElements, "Input" | "Label" | "Checkbox" | "Button">) => {
-  function ShadcnAutoInput(props: { field: string; label?: string }) {
+  const inputMakers = {
```

Comment:
@airhorns I switched from memoizing here and did the invocation outside. Is this okay?

Reply:
Yep this looks right to me! I think the `inputMakers` name is just a little confusing in that these arent the makers anymore, they are the made components. I might call this `inputComponents` or something like that instead!